src/tests/rememberMe.test.js

```diff
@@ -0,0 +1,46 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import localStorage from 'localStorage';
+import app from '../app';
+import mockData from './mockData';
+
+chai.use(chaiHttp);
+chai.should();
+
+const rememberMe = () => {
+  describe('Remember user who ckecked to be remembered ', () => {
+    it('it should return 200 when profile information strored in cookies successfull  ', (done) => {
+      chai
+        .request(app)
+        .get('/api/v1/remembered')
+        .end((err, res) => {
+          expect(res.statusCode).to.equal(200);
+        });
+      done();
+    });
+
+    it('it should return 200 when profile information strored in cookies successfull  ', (done) => {
+      chai
+        .request(app)
+        .get('/api/v1/remembered')
+        .end((err, res) => {
+          expect(res.statusCode).to.equal(200);
+          expect(res.body.data).to.be.an('object');
+        });
+      done();
+    });
+
+    it('it should return 500 you are not in the database ', (done) => {
```

Comment:
I think `404` is a correct status for a not found user case.

Reply:
That mistake has been resolved, you check

Reply:
> @EugeneMunya me also I get different results
> <img alt="Screen Shot 2020-03-03 at 16 50 59" width="1240" src="https://user-images.githubusercontent.com/50768983/75787402-58282380-5d6f-11ea-9fff-fd88ec3a461d.png">
The issue was the time it takes to update cookie storage and I manage this by storing the user information in sessions check how I did it